postinstall.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+if [ -n "$(node lib/nodegit.js 2>&1 | grep libstdc++)" ]; then
+	echo "[ERROR] Seems like you the latest libstdc++ is missing on your system!"
```

Comment:
A little grammatical error. Should read `"[ERROR] Seems like the latest libstdc++ is missing on your system!"